bin/main.rb

```diff
@@ -1,3 +1,68 @@
 #!/usr/bin/env ruby
+# rubocop:disable Metrics/PerceivedComplexity
+# rubocop:disable Metrics/CyclomaticComplexity
 
-print 'Hello world!'
+puts 'Welcome to Ruby Tic-Tac-Toe'
+
+# welcome to first player
+puts 'Welcome Player 1 please input x or o to choose your icon'
+player1 = gets.chomp
+
+player2 = player1 == 'x' ? 'o' : 'x'
+board = [
+  [' ', ' ', ' '],
```

Comment:
- [ ] You should ask the players for their names.

![image](https://user-images.githubusercontent.com/61479648/91335179-6dbe5900-e7a6-11ea-9ca7-6a8f1c299bda.png)